themes/codefor-theme/layouts/projekte/single.html

```diff
@@ -1,172 +1,74 @@
-                </div>
-              </div>
-            </div>
+          {{ if .Params.status }}
+          <p>Status des Projektes {{ .Title }}: {{ .Params.status }}</p>
+          {{ end }}
+          {{ if (and (isset .Params "imgname") (fileExists (printf "static/projects/%s" .Params.Imgname))) }}
+          <div class="d-flex align-items-center">
+              <img class="img-fluid mx-auto" src="/projects/{{ .Params.Imgname }}" alt="Screenshot {{ .Title }}">
           </div>
+          {{ end }}
+        </div>
+      </div>
 
-
-          <div class="card">
-            <div class="card-header px-0" id="heading-participate-04">
-              <div class="d-flex justify-content-between">
-                <button class="btn btn-link text-dark collapsed pl-0" type="button" data-toggle="collapse" data-target="#collapse-participate-04" aria-expanded="true" aria-controls="collapse-participate-04">
-                  <h3>Projektstatus</h3>
-                </button>
-                <button class="collapsed btn btn-close" type="button" data-toggle="collapse" data-target="#collapse-participate-04" aria-expanded="true" aria-controls="collapse-participate-04">
-                  <img class="img-fluid" src="/img/close-icon.svg">
-                </button>
-              </div>
-            </div>
-            
-            <div id="collapse-participate-04" class="collapse" aria-labelledby="heading-participate-04" data-parent="#accordionCode">
-              <div class="card-body py-0">
-                <div class="row justify-content-center">
-                  <div class="col-20 py-4">
-                    <ul class="list-unstyled mb-5">
-                      <div class="lab-icon d-flex justify-content-center">
-                        {{.Params.status }}
-                      </div>
-                    </ul>
-                  </div>
-                </div>
+      <div class="row">
+        <div class="col">
+          {{ partial "headline-brackets-h2.html" (dict "context" . "txt" "Projekt-Mitglieder & Labs" "color" "red" ) }}
+          <ul class="list-inline">
+            {{ range .Params.Collaborators }}
+            <li class="d-flex justify-content-center p-2">
+              <strong>{{ .name }}</strong>
+              {{ $name := .name }}
+              {{ range .links }}
+                {{ if in .url "github" }}
+                <a href="{{ .url }}"><img class="ext-link-icon mx-2" src="/img/github.svg" alt="Link zum GitHub Account von {{ $name }}"></a>
+                {{ else if in .url "twitter" }}
+                <a href="{{ .url }}"><img class="ext-link-icon mx-2" src="/img/twitter.svg" alt="Link zum Twitter Account von {{ $name }}"></a>
+                {{ else }}
+                <a href="{{ .url }}">{{ .name }}</a>
+                {{ end }}
+              {{ end }}
+            {{ end }}
+            </li>
+            </ul>
+            <div class="mb-5">
+              <div class="project-labs-list">
```

Comment:
I think the bootstrappy way here would be:
```suggestion
              <div class="d-flex justify-content-evenly">
```
I'm not a big fan of utility classes personally but at least this would be consistent with the rest of the code

Reply:
I'll also remove the class from `base.sass`